tests/ConnectionTest.php

```diff
@@ -715,5 +715,20 @@ public function testAllowsRetrievalOfSubstringsIndependentOfPlatform(): void
             self::assertEquals($expectedValue, $actualValue);
         }
     }
+
+    /**
+     * @throws \Artemeon\Database\Exception\QueryException
+     * @throws TableNotFoundException
+     */
+    public function testHasTable(){
+        $tableName = self::TEST_TABLE_NAME;
+        $connection = $this->getConnection();
+        $this->assertTrue($connection->hasTable($tableName));
+
+        $this->expectException(TableNotFoundException::class);
+        if(!$connection->hasTable('table_does_not_exist')) {
+            throw new TableNotFoundException('Table not found: ',self::TEST_TABLE_NAME);
```

Comment:
warum wirfst du hier eine Exception? ich hätte einfach nur ein 
```suggestion
        $this->expectException(TableNotFoundException::class);
        $connection->hasTable('table_does_not_exist');
```
erwartet

Reply:
Ich habe hierzu eben eine Korrektur gepusht. 
Diese Korrektur hatte ich prinzipiell schon gestern gehabt, aber der ist wegen eines Branchwechsel erst in den Stash gewandert. Ich vergaß, diesen anschließend wieder zurück zu holen und zu commiten. 

Demnach ist der Test insofern korrekt, da ich die Exception nicht innerhalb der hasTable Methode werden kann, da sonst alle andern Testcases fehlschlugen, da von den Tests ein Bool erwartet wurde, um eventuell eine Tabelle o.ä. anzulegen, soweit dieser nicht vorhanden ist. Daher musste ich den throw aus der hasTable rausnehmen und um den Methodenaufruf packen, damit die Exception dennoch richtig geworfen wird, aber die testcases dadurch nicht abschmieren.

Reply:
also die Methode `hasTable` sollte aus meiner Sicht keine Exception werfen sondern nur true oder false zurückgeben. Die Exception würde ich nur bei `getColumnsOfTable` und `getTableInformation` einbauen. Aber ansonsten sieht das mit der `TableNotFoundException` gut aus 👍